app/javascript/src/conversion-tracker.js

```diff
@@ -28,9 +28,10 @@ export default class {
 
   // Indicates if the passed date (represented as a string from localStorage) has expired based on daysToLive
   expired (dateString) {
-    const date = Date.parse(dateString)
-    const expiresAt = new Date(date.getTime())
-    expiresAt.setDate(expiresAt.getDate() + this.daysToLive)
+    const date = new Date(Date.parse(dateString)) // => Tue Jun 09 2020 14:33:59 GMT-0400 (EDT)
+    let expiresAt = new Date()
+    expiresAt.setTime(date.getTime())
+    const setAt = expiresAt.setDate(expiresAt.getDate() + this.daysToLive)
```

Comment:
This needs a look (and the comment can be removed) but I am pretty sure this is what we want. We may be able to make it a bit cleaner but this works.